src/ext/legit/peek-legit.lisp

```diff
@@ -320,14 +320,15 @@ Notes:
 (define-command peek-legit-select () ()
   (alexandria:when-let ((file (get-matched-file)))
     (quit)
-    (alexandria:when-let
+    (alexandria:if-let
         ((buffer (or (and (uiop:file-exists-p file)
                           (find-file-buffer file))
                      (find-file-buffer
                       (merge-pathnames
                        (lem-core/commands/project:find-root (buffer-filename))
                        file)))))
-      (switch-to-buffer buffer))))
+      (switch-to-buffer buffer)
+      (message "File ~a doesn't exist." file))))
```

Comment:
Elsewhere, we use editor-error to display errors.
Is it possible to do that here as well?

Reply:
Done :+1: